addons/test_mail/static/tests/activity.test.js

```diff
@@ -554,6 +554,57 @@ test("activity view: activity widget", async () => {
     await assertSteps(["action_feedback_schedule_next", "serverGeneratedAction"]);
 });
 
+test("activity widget: cancel an activity from the widget", async () => {
+    const [mailActivityId] = pyEnv["mail.activity"].search([["state", "=", "planned"]]);
+    const [mailActivityTypeId] = pyEnv["mail.activity.type"].search([["name", "=", "Email"]]);
+    pyEnv["res.users"].write([serverState.userId], {
+        activity_ids: [mailActivityId],
+        activity_type_id: mailActivityTypeId,
+    });
+    onRpc("unlink", (params) => {
+        step(`${JSON.stringify(params)}`);
```

Comment:
I'd check the route (contains the model and the method) and the args (for the correct id) individually here, then `expect.step('unlink')` instead of the entire JSON, and at the end of the test verify that step

(no need to check the kwargs/context here, it's the same for all rpc and not related to this test :)

Reply:
Hello @abd-msyukyu-odoo :wave: ,
Done with the changes :+1: